### DISCUSSION TITLE
#抽青衣歪了11号，望不周知。

### DISCUSSION THREAD

**Discussion by HKFDBHZN** [share121/inter-knot#2366] [其他]
啊，本来胜券在握，结果......我的青衣啊QAQ    

还发现我朱鸢伤害甚至不如别人的安东...   

我还是回到我的崩铁吧。

**Comment by GanlinSama** [comment]
抽青衣专武歪了11号专武<img src="https://img-static.mihoyo.com/communityweb/upload/fcb0d849f766bb544445a7ac0d5b9677.png" alt="妮可_生气" width="75" >

**Comment by YuTinMIn** [comment]
我一发10连歪了珂蕾妲……但是！我没有垫就出了，而且后来不到10抽有出了！
![微信图片_20240816201915](https://github.com/user-attachments/assets/5364c843-c292-4967-a354-705067942d54)
![微信图片_20240816201919](https://github.com/user-attachments/assets/76de5621-6514-446d-91c9-bdbd7f6c4d0b)
